.gitignore

```diff
@@ -11,6 +11,7 @@ terraform/**/.terraform.lock.hcl
 terraform/**/.terraform.tfstate.lock.info
 terraform/**/inventory.yaml
 terraform/**/plan.zip
+terraform/aws/plan.zip/aws_cluster_data.yaml
```

Comment:
Usually plan.zip is a file not a folder

Reply:
Ahh, good spot.  This is a typo, will fix.

Reply:
Fixed